### PR TITLE
ttnctl/util.GetConfigFile: check if config exists

### PR DESCRIPTION
**Pull request by rvolosatovs**
this prevents situations, when the variable is defined(or, for example, config flag is set), config file does not exist, but still is used. This PR makes sure `ttnctl` fallsback to other options for the config file and proper value is shown.

consider
```
1 b1101@atom ~ (git)-[atom] % ls ~/.config/ttn*                                                                                                                                                                                                                                                                                                                                                                                       
/home/b1101/.config/ttn-bridge.json
b1101@atom ~ (git)-[atom] % ttnctl -d user login XXX
 DEBUG Using config                             config file=/home/b1101/.config/ttnctl/config.yml data dir=/home/b1101/.local/share/ttnctl
```

### CODE REVIEW

File: ttnctl/util/config.go [hunk 1]
```diff
@@ -28,17 +28,31 @@ const (
...
+	if home != "" {
+		home = path.Join(home, ".ttnctl.yml")
+	}
+
+	for _, file := range []string{
+		flag,
+		xdg,
+		home,
+	} {
+		if len(file) > 0 {
```
Comment (htdvisser) [inline ttnctl/util/config.go]: `file != ""`

File: ttnctl/util/config.go [hunk 2]
```diff
@@ -28,17 +28,31 @@ const (
...
+			if _, err := os.Open(file); !os.IsNotExist(err) {
```
Comment (htdvisser) [inline ttnctl/util/config.go]: Use `os.Stat` instead of `os.Open` and check for `nil` error

File: ttnctl/util/config.go (hunk 1 above)
Comment (rvolosatovs) [inline ttnctl/util/config.go]: 1db1f0d

File: ttnctl/util/config.go (hunk 2 above)
Comment (rvolosatovs) [inline ttnctl/util/config.go]: 1db1f0d

**Pull request merged**